modules/trusted-firmware-m/tfm_boards/common/attest_hal.c

```diff
@@ -132,7 +143,15 @@ enum tfm_plat_err_t tfm_plat_get_boot_seed(uint32_t size, uint8_t *buf)
 	if (nrf_err != NRF_CC3XX_PLATFORM_SUCCESS) {
 		return TFM_PLAT_ERR_SYSTEM_ERR;
 	}
-
+#elif defined(CONFIG_PSA_NEED_CRACEN_KMU_DRIVER)
+	if( !boot_seed_set) {
+		psa_generate_random(boot_seed,32);
+		boot_seed_set = 1;
+	}
+	memcpy(buf,boot_seed, sizeof(uint8_t) * 32);
+#else
+	return TFM_PLAT_ERR_SYSTEM_ERR;
+#endif
```

Comment:
My gut feeling says there must be a better way to do this. @frkv do you have any thoughts?

Reply:
Build failure. `#error`